students/Chris_Kenyon/Lesson_2/PrintGrid.py

```diff
@@ -0,0 +1,66 @@
+###Lesson 2 making a program for creating grids of various sizes###
+
+#set values for simplicity
+p = "+"
+m = "-"
+s = " "
+v = "|"
+
+###Part 1: Brute force###
+#create a 2x2 grid of pre-specified size using print commands
+print(p + (m*4) + p + (m*4) + p)
+for i in range(4):
+	print(v + (s *4) + v + (s*4) + v)
+print(p + (m*4) + p + (m*4) + p)
+for i in range(4):
+	print(v + (s *4) + v + (s*4) + v)
+print(p + (m*4) + p + (m*4) + p)
+
+###Part 2: Making a basic function###
+#create a 2x2 grid of defined size using a function
+
+def print_grid(GridSize):
+	"""Create a 2x2 grid with with grid size determined from function argument"""
+	print(p + (m*GridSize) + p + (m*GridSize) + p)
```

Comment:
your indentation is off. you want 4 spaces exactly, you can usually configure your editor to do that.

Reply:
do you mean check the Tab option setting "replace by space" and set to 4?  If so that is what I have going forward.

Reply:
the setting will depend on an editor, but that sounds right